internal/pkg/service/stream/storage/statistics/provider_test.go

```diff
@@ -356,12 +426,22 @@ func TestStatisticsProviders(t *testing.T) {
 						RecordsCount:     100,
 						UncompressedSize: 100,
 						CompressedSize:   100,
+						StagingSize:      100,
 					},
 				}, stats)
 			},
 		},
 	}
 
+	l1Cache, err := cache.NewL1Cache(d.Logger(), statsRepo)
```

Comment:
Why do we use cache when we use `etcd` which should suite our needs? I would expect that the `etcd` should handle the requests that we are sending on it.

Reply:
In the test, there are tested 3 ways to get result, from etcd/repo, L1 and L2 cache:
https://github.com/keboola/keboola-as-code/blob/bef790a6be65e7b9164455a148aa5eac99f9086b/internal/pkg/service/stream/storage/statistics/cache/cache.go#L1-L7

https://github.com/keboola/keboola-as-code/blob/bef790a6be65e7b9164455a148aa5eac99f9086b/internal/pkg/service/stream/storage/statistics/cache/l1.go#L20-L29

https://github.com/keboola/keboola-as-code/blob/bef790a6be65e7b9164455a148aa5eac99f9086b/internal/pkg/service/stream/storage/statistics/cache/l2.go#L15-L25

Reply:
Just to be sure, this will be adopted in documentation.